test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -13,6 +13,9 @@ define([
 
     const JobCommChannel = JobComms.JobCommChannel;
 
+    const TEST_JOB_ID = 'someJob',
```

Comment:
The majority of the changes in this file are converting the strings/arrays used in tests to use these two constants.